# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/faanidct.c
# Commit: 186e2a334f2d
# Full Hash: 186e2a334f2deb37b29f44b28bc8c83babeeb9a4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-09 09:37:17
# Regressor Bug: 1877221
# File Overlap Count: 1
# Description:
#   Bug 1877221 - Pull in new ffmpeg code, reapply patch. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D199909
# ==============================================================================

diff -r c5920de78d16 -r 186e2a334f2d media/ffvpx/libavcodec/faanidct.c
--- a/media/ffvpx/libavcodec/faanidct.c	Thu Feb 08 10:39:53 2024 +0000
+++ b/media/ffvpx/libavcodec/faanidct.c	Thu Feb 08 10:39:54 2024 +0000
@@ -20,6 +20,7 @@
  */
 #include "faanidct.h"
 #include "libavutil/common.h"
+#include "libavutil/emms.h"
 
 /* To allow switching to double. */
 typedef float FLOAT;